src/eduid_webapp/signup/views.py

```diff
@@ -51,12 +51,12 @@
 def get_config():
     tou_url = current_app.config.get('TOU_URL')
     try:
-        r = requests.get(tou_url)
+        r = requests.get(tou_url, verify=False)
```

Comment:
Why don't we want to verify the certificate? Seems wrong.

Reply:
Jesus, Sorry, and thanks for spotting it, I did not mean to commit that, I have it because in my environment the certificates are self-signed